runelite-client/src/main/java/net/runelite/client/ui/SplashScreen.java

```diff
@@ -131,6 +132,7 @@ protected Color getSelectionForeground()
 		timer.start();
 
 		setVisible(true);
+		setDefaultCloseOperation(DO_NOTHING_ON_CLOSE);
```

Comment:
Should the previous call on line 76 be updated instead? https://github.com/runelite/runelite/blob/561a28ab8c38336b5998812409979bfb91169df7/runelite-client/src/main/java/net/runelite/client/ui/SplashScreen.java#L76